### PR TITLE
R code refactor

### PR DESCRIPTION
**Pull request by PavelBal**

### CODE REVIEW

File: shared/ClrFacade/ClrFacade.csproj [hunk 1]
```diff
@@ -26,6 +26,6 @@
   <PackageReference Include="R.NET" Version="1.8.1" />
   </ItemGroup>
   <Target Name="PostBuild" AfterTargets="PostBuildEvent">
-    <Exec Command="xcopy /Y $(TargetDir)*.dll $(SolutionDir)..\inst\libs\&#xD;&#xA;xcopy /Y $(SolutionDir)RSharp.runtimeconfig.json $(SolutionDir)..\inst\libs\" />
+    <Exec Command="xcopy /Y $(TargetDir)*.dll $(SolutionDir)..\inst\lib\&#xD;&#xA;xcopy /Y $(SolutionDir)RSharp.runtimeconfig.json $(SolutionDir)..\inst\lib\" />
```
Comment (PavelBal) [inline shared/ClrFacade/ClrFacade.csproj:29]: Here and in some other files, I have changed the `inst\libs\` to `inst\lib\`

File: shared/ClrFacade/RDotNetDataConverter.cs [hunk 2]
```diff
@@ -587,7 +587,7 @@ private SymbolicExpression convertObject(object obj)
 
    private Function _createClrS4Object;
 
-   public Function CreateClrS4Object => _createClrS4Object ??= _engine.Evaluate("invisible(rSharp::getCurrentConvertedObject)").AsFunction();
+   public Function CreateClrS4Object => _createClrS4Object ??= _engine.Evaluate("invisible(rSharp:::.getCurrentConvertedObject)").AsFunction();
```
Comment (PavelBal) [inline shared/ClrFacade/RDotNetDataConverter.cs:590]: Fixes https://github.com/Open-Systems-Pharmacology/rSharp/issues/45

**Comment by PavelBal** [comment]
- `inst\libs\` renamed to `inst\lib\` to stay consistent with the ospsuite-r package.
- Fixes https://github.com/Open-Systems-Pharmacology/rSharp/issues/45
- Removed the folder `rdotnet` as it was empty

- Please confirm that these files are not required:

- [ ] https://github.com/Open-Systems-Pharmacology/rSharp/blob/develop/INSTALL
- [ ] https://github.com/Open-Systems-Pharmacology/rSharp/blob/develop/cleanup
- [ ] https://github.com/Open-Systems-Pharmacology/rSharp/blob/develop/configure

**Comment by rwmcintosh** [comment]
I'll be removing cleanup and configure today once I make the disconnected Linux build work again. I have no idea about INSTALL. Isn't it documentation? It should probably be rewritten to match whatever process we come up with and be addressed in #19

**Review by rwmcintosh — APPROVED** [review]

**Pull request merged**